gravitee-am-gateway/gravitee-am-gateway-handler/gravitee-am-gateway-handler-account/src/test/java/io/gravitee/am/gateway/handler/account/resources/AccountFactorsEndpointHandlerTest.java

```diff
@@ -87,13 +86,16 @@ public class AccountFactorsEndpointHandlerTest extends RxWebTestBase {
     @Mock
     RateLimiterService rateLimiterService;
 
+    @Mock
+    AuditService auditService;
```

Comment:
**todo**: In this Test class could be nice to check that tha auditService is called with the right type of Event IMO.

Reply:
done, service call verified.